src/components/Results.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import "./Results.css"
+
+const Results = (props) => {
+  console.log(props.results)
+  const originalUpload = props.results.pop()
+  const results = props.results
+
+  return (
+    <div className="resultsContainer">
+      <div className="uploadContainer">
+        <h2>Your Upload</h2>
+        <img className="uploadImage" src={originalUpload} alt='original upload' />
+      </div>
+      <div className="topMatchesContainer">
+        <h2>Your Top Matches</h2>
+        <div className="topImagesContainer">
+          { results.map((item) => {
+              return (
+              <figure>
+                <img className="resultImage" src={item.image_url} key={item.uid} confidence={item.confidence} alt={item.url} />
+                <figcaption>
+                  <a href={item.url}>{item.title}</a>
+                </figcaption>
+              </figure>
```

Comment:
Maybe make this another component?

Reply:
Is figcaption not preferred?

Reply:
I think the semantic HTML with `figure` and `figcaption` is really good actually. What I meant was putting all that and the `div` wrapper into some `Figure` component that you can just pass the `results` as a prop to and clean up the nested JSX.